Src/BootCamp.Chapter/CsvParsingException.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    public class CsvParsingException : Exception
```

Comment:
Not a fan of such exceptions. I try to give context to exceptions as much as possible. I would say that a file where it failed should be passed as argument.